policy/modules/services/policykit.fc

```diff
@@ -11,6 +11,7 @@
 # Systemd unit file
 /usr/lib/systemd/system/[^/]*polkit.*	--	gen_context(system_u:object_r:policykit_unit_t,s0)
 
+/usr/libexec/polkit-agent-helper-1	--	gen_context(system_u:object_r:policykit_auth_exec_t,s0)
```

Comment:
Perhaps the version should be handled with `.*` instead, like the other lines? I'm unsure since there seems to be a bunch of variance between distros.

Reply:
The fc here look a bit a mess.

On my debian unstable machine, the only file installed are `/usr/libexec/polkit-agent-helper-1` and `/usr/libexec/polkitd` (for version 0.105). The version in debian experimental (0.116) is still installing the same files in `/usr/lib/polkit-1`

I'm not sure if the other file contexts are still used anywhere by any distro

I've opened a [bug upstream](https://gitlab.freedesktop.org/polkit/polkit/-/issues/125) to officially move them to `/usr/libexec/polkit-1` so maybe we could wait until that bug is fixed upstream and carry the patch in debian only

Reply:
> I've opened a [bug upstream](https://gitlab.freedesktop.org/polkit/polkit/-/issues/125) to officially move them to `/usr/libexec/polkit-1` so maybe we could wait until that bug is fixed upstream and carry the patch in debian only

Sounds like a good plan.